wgengine/router/router_userspace_bsd.go

```diff
@@ -10,6 +10,7 @@ import (
 	"fmt"
 	"log"
 	"os/exec"
+        "runtime"
```

Comment:
not gofmt'ed?

Guess we don't have a GitHub Action for that.

Reply:
I did most of the editing on the newly created FreeBSD VM where I don't have the vim-go plugin nor other stuff installed. Formatted now.